skin/assets/styles/utils/_helpers.scss

```diff
@@ -19,41 +19,41 @@
 }
 
 // Rotate helpers
-.u-orientation--0 {
-  transform: rotate(0);
-}
+.u-orientation {
+  &--0 {
+    transform: rotate(0);
+  }
 
-.u-orientation--90 {
-  transform: rotate(90deg);
-}
+  &--90 {
+    transform: rotate(90deg);
+  }
 
-.u-orientation--180 {
-  transform: rotate(180deg);
-}
+  &--180 {
+    transform: rotate(180deg);
+  }
 
-.u-orientation--270 {
-  transform: rotate(-90deg);
+  &--270 {
+    transform: rotate(-90deg);
+  }
 }
 
 // Utility for responive videos
-.u {
-  &__embed-video-responsive {
-    position: relative;
-    display: block;
-    height: 0;
-    overflow: hidden;
-    padding-bottom: 56.25%;
+.u-embed-video-responsive {
+  position: relative;
+  display: block;
+  height: 0;
+  overflow: hidden;
+  padding-bottom: 56.25%;
 
-    &__video,
-    iframe { // stylelint-disable-line selector-no-type
-      position: absolute;
-      top: 0;
-      bottom: 0;
-      left: 0;
-      width: 100%;
-      height: 100%;
-      border: 0;
-    }
+  &__video,
```

Comment:
This is used somewhere in the boilerplate? It sounds... odd? If not we can just move this to wiki as an example.